HowToFIDO.md

```diff
@@ -977,3 +993,15 @@ a UVRA</span>](#signing-in-with-a-uvra).
 15. Historically, we used the term "resident key" to refer to what will
     be known as "discoverable credentials" in the upcoming WebAuthn L2
     and CTAP2.1 specs.
+
+16. This limitation may be solved if CTAP is implemented on the platform 
+    authenticators. CTAP will enable platform authenticators to securely connect
+    to other devices via local transports like BLE and enable bootstrapping the 
+    device. No off-the-shelf solution is available at the time of writing. 
```

Comment:
Do we want to include this in "how-to-fido"? Sounds like more of a TWG discussion than guidance for FIDO RPs?